lieer/remote.py

```diff
@@ -106,6 +106,8 @@ def __init__ (self, g):
     self.CLIENT_SECRET_FILE = g.credentials_file
     self.account = g.local.config.account
     self.dry_run = g.dry_run
+    if g.local.age:
+       self.query = "newer_than:{}m".format(g.local.age)
```

Comment:
This resets the existing query, it should be appended with an AND.

Reply:
Thank you for reminding me of this issue. I'll fix it in the next commit.